src/audio.cpp

```diff
@@ -4,13 +4,14 @@
 
 #include <math.h>
 
-#include "config-opentomb.h"
+#include "config-default/config-opentomb.h"
```

Comment:
Yes, that's wrong. There are platform dependent includes which won't work on another platform.